### PR TITLE
Implemted P pattern teleport

### PR DESCRIPTION
**Pull request by windwalker-888**
not tested on players

### CODE REVIEW

**Comment by Crowdedlight** [comment]
Just to clarify. 20 units is required for the base pattern, and then it extends outwards if above 20? So if you are less than 20, it will be a broken pattern?  
Does it fill up so even with 5 it somewhat appears like the right pattern, or does it fill from one side to another? 

I'll test it one of the coming days.

**Comment by windwalker-888** [comment]
> Just to clarify. 20 units is required for the base pattern, and then it extends outwards if above 20? So if you are less than 20, it will be a broken pattern?
> Does it fill up so even with 5 it somewhat appears like the right pattern, or does it fill from one side to another?
> 
> I'll test it one of the coming days.

it needs 20 people to do one "complete" shape. 
so yes, less than that its just... not "right" ;) 
it goes west to east, it dont fill up or streach out.

**Comment by Crowdedlight** [comment]
Reopenend. Spotted some issues that needs to be tested first. 

The ``_offset`` applies flat offset to x position and just offsets the position of the figure from where you clicked and doesn't scale up the figure with more distance between each position.

File: CrowsZA/functions/fn_scatterPatternOddPattern.sqf [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+private _tick = 0;
+
+// loop and build positions
+for "_k" from 1 to _amount do { 
+	
+	private _tmpI = 0;
+	private _tmpJ = 0;
+	
+	switch (_tick) do {
+		case 0:  { _tmpI = _i + 0  	 + 0 	  ;		_tmpJ = _j + 0 +	(10 * _row)};
```
Comment (Crowdedlight) [inline CrowsZA/functions/fn_scatterPatternOddPattern.sqf:36]: Doesn't have an offset, so the first unit is placed where you click, but the rest is offset in x direction and creates a figure ``_offset`` away.

**Pull request merged**